README.md

```diff
@@ -166,9 +167,8 @@ and carries no guarantee from The Above Projects about quality, suitability or a
 
 [GNU General Public License version 3](https://www.gnu.org/licenses/gpl-3.0.txt)
 
-Copyright (c) 2019-2021 Garmatin Oleksandr invizible.soft@gmail.com
-
-All rights reserved
+Copyright © 2019–2021 Garmatin Oleksandr and contributors
+invizible.soft@gmail.com
```

Comment:
Please do not change the copyright. If you disagree with it, unfortunately I cannot accept the merging of your pull requests.

Reply:
You need a CLA to have sole copyright and that was never the case. "and contributors" is accurate.

Reply:
This is my project anyway. So if you disagree, I can only suggest that you fork and develop your own application however you want.
I see no point in cooperation. You wanted to translate a project, but you really want to redo it. This is unacceptable to me. Sorry if this was not clear from the beginning.

Reply:
I have zero changes in your project. I am adding that as an acknowledgement of others.
Your project includes code from different projects, and has accepted contributions since then.
How you declare copyright has no bearing on what is covered by their copyright, because that is not how copyright works.
Just trying to help.

Reply:
Actually, the problem is not with the copyright. The problem is you are making changes that are not translation related. I appreciate your help, but your changes are not acceptable. I accept all contributors, but I accept only those changes that, in my opinion, are useful for the project. So you cannot force me to merge your changes.

Reply:
This is very useful, because it shows gratitude for the help you got along the way.
I am not trying to force you to do anything.
I make changes to work I have done if the changes make sense.
You seem to have an idea about copyright or my intentions that isn't shared by me.

If I spend the effort to fix the language, it really makes sense for me to also improve the landing page.
That is where my honor lies.

Reply:
I can accept your help if you only do the translation. If you disagree, I cannot merge both pool requests. I appreciate the help of the contributors. It is very important for me. But everyone must do their own thing, otherwise the project has no future.

Reply:
One PR has nothing to do with the other.
Most importantly mentioning Hosted Weblate is one of the terms of using that platform. Other than that I don't understand what "translation" is supposed to mean.
I offered to set up Weblate, translate some languages, fix up the source strings, and set up donations.
None of that is for me, all of it is free labour for you.
What I do is this sort of thing. I can't imagine how that isn't a fantastic deal for you, seeing as you don't want to spend the time doing it.
I am like that too, I spend my time fixing things, not redoing unimportant stuff.

You can do with this README PR exactly what you want, subject to the terms of the license.
The translation I have already made is subject to the same copyright, and that is shared on the terms of the license.
If having contributors is important to you, appreciating them by mentioning it in the copyright notice is a good gesture.
It changes absolutely nothing other than to send some goodwill out into the world.

Reply:
> One PR has nothing to do with the other.

My copyright in every InviZible's file. If you disagree with it, how can I use your work? It's not fair. I want to be fair.

> Most importantly mentioning Hosted Weblate is one of the terms of using that platform

Of course, I will mention it. Unfortunately, I didn't have time for this before.

> I offered to set up Weblate, translate some languages, fix up the source strings, and set up donations.

And I appreciate your work and your time. But if you did just that, there would be no questions.

> The translation I have already made is subject to the same copyright

Therefore, you can agree to my terms or not. So I can use your incredible work or not. The choice is simple.

Reply:
I don't disagree with your name being listed as the main copyright holder.
I don't disagree with you re-iterating this in every file.
I don't even disagree with your name listed as the copyright holder of every file. However, if it is in a translation you had no part in, that is an error.

If you take any contributions without a CLA, it means copyright is shared between more than one entity.
This is why the GPLv3+ protections mean anything. Nobody can effectively re-license into a weaker license, because everyone (every copyright holder) has to agree.
Nobody has the copyright to files only you wrote btw.

I didn't remove your copyright, I merely added "and contributors". That is accurate.
When I or anyone else contributes it is done as per the GPLv3+ terms.
You and anyone else can use it under those terms, but it doesn't transfer copyright to anyone.

Can you clarify what you think will happen if you merge this 
"Copyright © 2019–2021 Garmatin Oleksandr and contributors" line?

What I think will happen is nothing, and that it is a nice nod to the people that contributed, and that people will be more likely to contribute in the future because of it.

Reply:
> I don't even disagree with your name listed as the copyright holder of every file. However, if it is in a translation you had no part in, that is an error.

This is the reason I am unable to merge your pull requests.

> Can you clarify what you think will happen if you merge this

InviZible is my child. I feel responsible for its fate. If I add "and ...", it will mean that it is no longer mine, but some of our children. I know it sounds crazy, but I can't do it. So anyone can use, fork or help develop the project. And I appreciate that. But the original InviZible will only be one.

Reply:
OK, I get it. What you are thinking of is the name "InviZible". You will be the maintainer of it until you decide not to be, or ultimately die of old age.
Nobody can make the same project under the same name, but they can make a new project starting with the same code-base.

> A “contributor” is a copyright holder who authorizes use under this License of the Program or a work on which the Program is based. The work thus licensed is called the contributor's “contributor version”.

This is from the GPLv3+ license, where it details how contributors grant claims they could have made for patents to everyone that abides by the license.

The copyright isn't for the name, which is the real project, but rather the license to use it under GPLv3+.
Everyone that has a copyright claim, agrees to the GPLv3+ license when contributing.
The license is protected by everyone that has a copyright claim to their parts of the codebase.
If it was all 100% done by you, you could effectively re-license it into a proprietary product.
That can still happen, but now everyone holding a copyright claim to the source code will have to agree.

That you care about the product is all wall and good, and I would definitely think of that as a positive message to send.
What not including "and contributors" does is send the message that you don't care about others.
I can remove "and contributors" if you feel bad about it, but that doesn't actually change anything about the copyright others hold for their parts of the project. Those parts are available to you under GPLv3+, just as your parts are available to others under the same license :)

Reply:
> OK, I get it. What you are thinking of is the name "InviZible". You will be the maintainer of it until you decide not to be, or ultimately die of old age. Nobody can make the same project under the same name, but they can make a new project starting with the same code-base.
> 
> > A “contributor” is a copyright holder who authorizes use under this License of the Program or a work on which the Program is based. The work thus licensed is called the contributor's “contributor version”.
> 
> This is from the GPLv3+ license, where it details how contributors grant claims they could have made for patents to everyone that abides by the license.
> 
> The copyright isn't for the name, which is the real project, but rather the license to use it under GPLv3+. Everyone that has a copyright claim, agrees to the GPLv3+ license when contributing. The license is protected by everyone that has a copyright claim to their parts of the codebase. If it was all 100% done by you, you could effectively re-license it into a proprietary product. That can still happen, but now everyone holding a copyright claim to the source code will have to agree.
> 
> That you care about the product is all wall and good, and I would definitely think of that as a positive message to send. What not including "and contributors" does is send the message that you don't care about others. I can remove "and contributors" if you feel bad about it, but that doesn't actually change anything about the copyright others hold for their parts of the project. Those parts are available to you under GPLv3+, just as your parts are available to others under the same license :)

Greetings,
I take the liberty to intervene in this discussion, I personally (and still) made the translation of the application in French (entirely, and I regularly make the translation of the new added elements), 
I feel concerned, because I am a (very small) contributor to the InviZible Pro Project and I approve of the developer of the application in the first place, let me explain:

Demanding a recognition for having worked on a free application, without advertising and developed by a single dev seems to me to be a pettiness, personally if I made this "work" of translation it is above all for the pleasure to offer to the French-speaking users of InviZible Pro the ease of use of a translated application (the French are globally null in English. ..in foreign languages rather^^), not to get my name in the application, moreover, the developer works hard to make this application is quality, it's his job, (and it's a huge coding job behind it) not ours, we must be honest.

Don't see the aggression towards you, it's not the case, from France with love.

Reply:
I isn't petty, but none of the small contributors are doing it (me included).
You don't have your name in the translation (and neither do I).

Read [what I actually put in](https://github.com/Gedsh/InviZible/pull/112/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R3) and you would notice it says the name of the author, first thing.
Nobody else mentioned there.

None taken, your issue is purely with how copyright works.
I can explain copyright or the psychology behind appearing charitable if you want.